rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -733,12 +734,11 @@ private FederationChangeResponseCode commitPendingFederation(PendingFederation c
         // set proposed federation
         Federation proposedFederation = buildFederationFromPendingFederation(currentPendingFederation);
         provider.setProposedFederation(proposedFederation);
+        setNewActiveFederationCreationBlockHeight();
```

Comment:
Should this be done at this point? Shouldn't we set this value after the proposed federation is validated? If the validation fails we'd end up with an invalid value here, right?

Reply:
we discussed this when developing phase1, the idea was that the activation period remain the same. But yes it makes sense to do it after the validation.

Reply:
updated